tests/estimation/test_get_estimation.py

```diff
@@ -0,0 +1,122 @@
+"""
+get_estimation.py::get_estimation
+
+We test all the benchmark_mediation estimators for a certain tolerance.
+The test is skipped if estimator has not been implemented yet,
+i.e. all outputs are NaN or NotImplementedError is raised.
+The test xfails for any other wierd behavior.
+
+We pinpoint that :
+- DML_huber is not working, RRuntimeError is raised
```

Comment:
specify that it is when dim_x=1, and hence, add tests with a different dimension for dim_x (with parameterize?)

Reply:
+ to explore and fix @judithabk6

Reply:
I will add such tests in a future PR